metadeploy/api/tests/test_views.py

```diff
@@ -68,20 +68,39 @@ def test_multi_tenancy(self, client, extra_site):
 
 
 @pytest.mark.django_db
-class TestObtainTokenView:
-    def test_multi_tenancy(self, anon_client, user_factory, extra_site):
+class TestResetTokenView:
+    def test_reset_token(self, client):
         url = reverse("token")
-        user = user_factory()
-        data = {"username": user.username, "password": "foobar"}
-        token1 = Token.objects.create(user=user)
-        with override_current_site_id(extra_site.id):
-            token2 = Token.objects.create(user=user)
+        user = client.user
+        old_token = Token.objects.create(user=user)
+        response = client.get(url)
+
+        tokens = Token.objects.filter(user=user)
+        # Refreshing should not add another token
+        assert tokens.count() == 1
+
+        new_token = tokens[0]
+        assert response.data.get("token") == new_token.key
+        assert old_token.key != new_token.key
```

Comment:
Let's assert that the token's are completely different instances:
```suggestion
        assert new_token is not old_token
```